drivers/console/Kconfig

```diff
@@ -34,7 +34,7 @@ config CONSOLE_HANDLER
 
 config CONSOLE_INIT_PRIORITY
 	int "Console init priority"
-	default 95 if USB_UART_CONSOLE || UART_MUX
+	default 95 if UART_MUX
```

Comment:
> So what has changed, since console can now be initialized at `60` instead ?
> This commit only changed the symbol used: [d8bc37b](https://github.com/zephyrproject-rtos/zephyr/commit/d8bc37beeb4d1eb4fdc47f132c15915f73a9ef50)
> but the default value is still `50`
> `SERIAL_INIT_PRIORITY` defaults to `KERNEL_INIT_PRIORITY_DEVICE` default == `50`

CDC ACM UART driver has been reworked a bit and does not block when USB device support is not initialized, (behaves like regular UART).

>Or should the setting be promptless in certain cases ?

I am in favor of this option.